src/components/BooksTap/Book.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Book = ({ book }) => (
+  <Item>
+    <Img src={book.image} />
+    <Wrapper>
```

Comment:
```suggestion
    <UserInfo>
```
이름이 내용을 반영하고 있으면 더 좋을거 같습니다

Reply:
확인하고 변경하였습니다!!